pattoo/api/portal/react/src/DatapointHead.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+class DatapointHead extends React.Component {
+    render() {
+        return (
+            <thead className="bg-gray-600 text-white">
```

Comment:
Where are the comments on what you are doing and why? This is important for future developers.